core/backend/src/IModelHost.ts

```diff
@@ -457,9 +457,14 @@ export class IModelHost {
       WipRpcImpl,
       DevToolsRpcImpl,
       Editor3dRpcImpl,
-      NativeAppRpcImpl,
     ].forEach((rpc) => rpc.register()); // register all of the RPC implementations
 
+    if (BackendIpc.isValid) {
+      [
+        NativeAppImpl,
+      ].forEach((ipcHandler) => ipcHandler.register());
+    }

```

Comment:
nitpick but why not simply `NativeAppImpl.register()`?

Reply:
only if we end up having more built-in `IpcInterfaces`. I'm not sure i like this here anyway.